src/ResourceManager/ApiManagement/Commands.ApiManagement/help/New-AzureRmApiManagementApiRelease.md

```diff
@@ -0,0 +1,180 @@
+---
+external help file: Microsoft.Azure.Commands.ApiManagement.ServiceManagement.dll-Help.xml
+Module Name: AzureRM.ApiManagement
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.apimanagement/new-azurermapimanagementapirelease
+schema: 2.0.0
+---
+
+# New-AzureRmApiManagementApiRelease
+
+## SYNOPSIS
+Creates an API Release of an API Revision
+
+## SYNTAX
+
+```
+New-AzureRmApiManagementApiRelease -Context <PsApiManagementContext> -ApiId <String> -ApiRevision <String>
+ [-ReleaseId <String>] [-Note <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The **New-AzureRmApiManagementApiRelease** cmdlet creates an API Release for an API Revision in API Management context.
+
+## EXAMPLES
+
+### Example 1: Create an API Release for an API Revision
+```powershell
+PS C:\>$ApiMgmtContext = New-AzureRmApiManagementContext -ResourceGroupName "Api-Default-WestUS" -ServiceName "contoso"
+PS C:\>New-AzureRmApiManagementApiRelease -Context $ApiMgmtContext -ApiId "echo-api" -ApiRevision "2" -ReleaseId "echo-api-release" -Note "Releasing version 2 of the echo-api to public"
```

Comment:
@solankisamir please make sure your examples have a sample output from when the command is executed to give users a better idea of what they can expect to be returned from the cmdlet invocation

Reply:
can you provide a sample of what you are saying? I am not clear.

---
In reply to: [188768556](https://github.com/Azure/azure-powershell/pull/6189#discussion_r188768556) [](ancestors = 188768556)

Reply:
@solankisamir see the examples for [`Get-AzureRmResource`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmResource.md#examples) -- all of the examples of a sample output of what the user would see when executing the cmdlet